### PR TITLE
Added tests for dist

### LINKED ISSUE #1196: Tests for dist

### PR DESCRIPTION
**Pull request by blond**
Resolved #1196

### CODE REVIEW

**Comment by blond** [comment]
Test for `tick` block failed because [tick_start_auto](https://github.com/bem/bem-core/blob/v2/common.blocks/tick/_start/tick_start_auto.vanilla.js) is included to code.

Can we add `tick.stop();` in [beforeEach](https://github.com/bem/bem-core/blob/v2/common.blocks/tick/tick.spec.js#L8-L10) to fix it?

File: test/dist/assets/test.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                    });
+            }());
+        </script>
+${ scripts }
+        <script>
+            (function() {
+                var global = this;
+
+                modules.require(["jquery", "mocha", "spec"], function($, mocha) {
+                    if (global.mochaPhantomJS) {
```
Comment (tadatuta) [inline test/dist/assets/test.html]: `(global.mochaPhantomJS || mocha).run(done);`

File: test/dist/assets/test.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                        provide();
+                    });
+            }());
+        </script>
+${ scripts }
+        <script>
+            (function() {
+                var global = this;
+
+                modules.require(["jquery", "mocha", "spec"], function($, mocha) {
```
Comment (tadatuta) [inline test/dist/assets/test.html]: `"` -> `'`

File: test/dist/build-fixtures.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        fs.writeFileSync(path.join(fixturesDir, platform + '.html'), html, 'utf-8');
+    });
+});
+
+function buildHTML(platform, specs) {
+    return asset
+        .replace(/\${ platform }/g, platform)
+        .replace(/\${ scripts }/g, specs.map(function (url) {
+            return '        <script src="../../../' + url + '"></script>';
+        }).join('\n'));
```
Comment (tadatuta) [inline test/dist/build-fixtures.js]: `EOL`?

**Comment by blond** [comment]
:up:

**Pull request merged**